azure-pipelines.yml

```diff
@@ -68,6 +68,6 @@ jobs:
       conda create -n blarg1 -yq python=2.7
       conda create -n blarg3 -yq python=3.7
       conda create -n blarg4 -yq python nomkl numpy pandas svn
+      source /usr/share/miniconda/bin/activate
```

Comment:
I don't think this line is necessary , is it?

Reply:
I did not expect this, but it is required. 

Seems like all the dependencies are installed into the base environment and then the next line call py.test with absolute path instead of previously activating the environment. The shellcheck command that is installed has to be in path, which can be done via the activate line or something like `export PATH="/usr/share/miniconda/bin/activate:$PATH"`. If that is not done an older shellcheck version is found, which is part of the ubuntu 16.04 image use for azure ci.

What is preferred here? activate/path? .... also a switch to the ubuntu shellcheck is possible, which would not require that line at all and also would not require shellcheck in the main channels.

Reply:
Could you try to just call ``pytest`` then? I think the entire path to pytest is not necessary as well

Reply:
Ok. Lets see if that works :)

Reply:
It worked! :tada: haha

Could you please just solve the conflicts? after that I think it should be good to go!

Reply:
ok. rebased the commits.